File: Palindrome String[25-August-2023]/solution.py
#  >> PYTHON CODE <<

class Solution:
	def isPalindrome(self, S):
	    low,high=0,len(S)-1
	    
	    while(low<high):
	        if(S[low]!=S[high]): return 0
	        
	        low+=1
	        high-=1
	       
	    return 1

File: Palindrome String[25-August-2023]/Solution.cpp
//  >> C++ CODE <<

class Solution{
public:	
	
	
	int isPalindrome(string S)
	{
	    int low=0,high=S.length()-1;
	    
	    while(low<high)
	    {
	        if(S[low]!=S[high]) return 0;
	        
	        low++; high--;
	    }
	    
	        return 1;
	}

};